cloud-stream-custom-queue-names/src/main/java/com/solace/samples/spring/scs/GeneratedQueueNames.java

```diff
@@ -0,0 +1,37 @@
+package com.solace.samples.spring.scs;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.integration.StaticMessageHeaderAccessor;
+import org.springframework.integration.acks.AckUtils;
+import org.springframework.integration.acks.AcknowledgmentCallback;
+import org.springframework.messaging.Message;
+
+import com.solace.spring.cloud.stream.binder.util.SolaceAcknowledgmentException;
+
+import java.util.function.Consumer;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@SpringBootApplication
+public class GeneratedQueueNames {
+	private static final Logger log = LoggerFactory.getLogger(GeneratedQueueNames.class);
+
+	public static void main(String[] args) {
+		SpringApplication.run(GeneratedQueueNames.class, args);
+	}
+	
+	@Bean
+	public Consumer<Message<String>> uppercase() {
```

Comment:
let's make sure we concentrate on the custom queue names here. I'd suggest remove the manual ACK stuff.

Reply:
The only reason I included the ack and rejection action here is to demonstrate that the error queue (with custom name) receives the rejected events. I will remove this and just consume the events.